LanguageExt.Core/DataTypes/Recursive/Recursive.Prelude.cs

```diff
@@ -0,0 +1,451 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using LanguageExt.DataTypes.Recursive;
+
+namespace LanguageExt
+{
+    public static partial class Prelude
+    {
+
+        /// <summary>
+        /// Returns the base of the recursion
+        /// </summary>
+        /// <typeparam name="A"></typeparam>
+        /// <param name="t"></param>
+        /// <returns></returns>
+        public static Recursive<A> Return<A>(A a) => new Recursive<A>(a);
```

Comment:
Also `Yield` is a verb rather than noun, so for a constructor name it's not ideal.  And with that in mind `Recurse` is a bad choice also.  We need something like `Some`, `None`, `Left`, `Right`, something that gives an indication of the state of the `Recursive` once constructed.

The only things that come to mind are 'termination' synonyms: `Final`, `End`, `Last`, `Done`, etc.